jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/AsyncServletLongPollTest.java

```diff
@@ -202,12 +183,18 @@ public void destroy()
 
             await().atMost(5, TimeUnit.SECONDS).until(asyncContextRef::get, Matchers.notNullValue());
 
+            // Wait for the request on the server to become idle.
+            Thread.sleep(1000);
```

Comment:
I'm sure awaitility can be used to replace this `sleep`, possibly by waiting for the thread that executed the handler to have its status changed to WAITING.